src/Rotations.jl

```diff
@@ -1,5 +1,47 @@
+# we generate code in this module, so precompile where possible
+ VERSION >= v"0.4.0-dev+6521" && __precompile__(true)
+
 module Rotations
 
-# package code goes here
+using FixedSizeArrays
+using Quaternions  # TODO: Switch to the UnitQuaternions package? https://github.com/kam3k/UnitQuaternions.jl
+
+import Base: convert, mean, eltype, isnan, length, call, getindex, inv, .*, tuple
+import Quaternions.Quaternion
+
+src_files = ["rotation_types", "rotation_conversions", "euler_conversions", "type_methods", "rotate", "quaternion_ops", "math_funcs", "rot_covariances"]
+for file in src_files
+    #println("$(file).jl")
+    include("$(file).jl")
```

Comment:
I've turned 4 lines into 8 for you xD

On 9 May 2016 at 17:39, Chris Foster <notifications@github.com> wrote:

> In src/Rotations.jl
> <https://github.com/FugroRoames/Rotations.jl/pull/2#discussion_r62459089>:
>
> >  module Rotations
> >
> > -# package code goes here
> > +using FixedSizeArrays
> > +using Quaternions  # TODO: Switch to the UnitQuaternions package? https://github.com/kam3k/UnitQuaternions.jl
> > +
> > +import Base: convert, mean, eltype, isnan, length, call, getindex, inv, .*, tuple
> > +import Quaternions.Quaternion
> > +
> > +src_files = ["rotation_types", "rotation_conversions", "euler_conversions", "type_methods", "rotate", "quaternion_ops", "math_funcs", "rot_covariances"]
> > +for file in src_files
> > +    #println("$(file).jl")
> > +    include("$(file).jl")
>
> Personally I'd just include all the files here, the loop doesn't add much
> IMO.
>
> include("rotation_types.jl")# ...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/FugroRoames/Rotations.jl/pull/2/files/fb3a7882257b93afed97cfaa9bf56669bbf410dc#r62459089>
>